governance/newsletters/README.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Apereo Newsletters
+layout: default
+permalink: governance/newsletters/
+---
+
+# Newsletters
+
+[Subscribe][]
+
++ [May 2018][2018-05-17 newsletter]
++ [March/April 2018][2018-04-23 newsletter]
++ [February 2018][2018-02-16 newsletter]
++ [January 2018][2018-01-15 newsletter]
```

Comment:
Progress! :+1: 
Ideally news letters could be internally represented as `_posts` https://jekyllrb.com/docs/posts
Or as `_data` if they continue to be external https://jekyllrb.com/docs/datafiles/
This is a good start :+1:

Reply:
Using post would have an added benefit that it can automatically generate an RSS/Atom feed https://github.com/jekyll/jekyll-feed